lib/src/mt_sch.c

```diff
@@ -138,6 +140,13 @@ static int sch_tasklet_func(void* args) {
         continue;
       }
       ops = &tasklet->ops;
+      /* To wait for time sync, only start "cni" ops in the first 5 minutes. */
+      if (strcmp(ops->name, "cni")) {
```

Comment:
Seems like it a special WA, can we move it another PR? And this WA should applied only for some special case.

Reply:
> Seems like it a special WA, can we move it another PR? And this WA should applied only for some special case.

Resovled by adding wait for timesync condition of [commit](https://github.com/OpenVisualCloud/Media-Transport-Library/pull/295/commits/583c74174b10a3415f3b21e773c29bacac966fc8)